dev/integration_tests/web/web/index_with_blocked_service_workers.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE HTML>
+<!-- Copyright 2014 The Flutter Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. -->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+
+  <title>Web Test</title>
```

Comment:
Might be useful for the page title to reflect what test this is.

Reply:
Yikes, I didn't see this. I'm going to create a bug to improve the integration_tests for web, they're growing quite a bit.